grails-app/views/_menu/_navbar.gsp

```diff
@@ -61,9 +61,16 @@
                         <g:link controller="aggregation" action="show"
                                 title="${message(code:'de.iteratec.isocsi.aggregationChart', default:'Aggregation')}">
                             <i class="fas fa-chart-bar"></i>
-                                <g:message code="de.iteratec.isocsi.aggregationChart" default="Aggregation"/>
+                            <g:message code="de.iteratec.isocsi.aggregationChart" default="Aggregation"/>
                         </g:link>
                     </li>
+                    <!--<li class="${request.forwardURI.equals('/aggregationDev/show') ? 'active' : ''}"
+                        data-active-matches="/aggregationDev/?.*">
+                        <a href="${createLink(uri: '/aggregationDev/show')}">
+                            <i class="fas fa-chart-bar"></i>
+                            <g:message message="frontend.de.iteratec.osm.aggregation.title"/>
+                        </a>
+                    </li>-->
```

Comment:
This can be removed or am I missing something?

Reply:
This is the navigation element for the new Aggregation page that will be used if the new Aggregation page is complete.